app/models/shelter.rb

```diff
@@ -16,6 +17,14 @@ def self.order_by_number_of_pets
       .order("pets_count DESC")
   end
 
+  def self.reverse_alpha
+    Shelter.find_by_sql "select * from shelters order by name desc"
+  end
```

Comment:
To answer the question from the video, more often than not you are going to use the ORM of that language over using pure SQL. There are situations where SQL is going to be easier to use however, one that I can think of right now it sub-querying...